### ISSUE TITLE
management canister did file not found

### ISSUE THREAD

**Issue by letmejustputthishere** [dfinity/vscode-motoko#263]
this canister builds, and `.dfx/local/canisters/idl/aaaaa-aa.did` is being generated.
```
import Management "ic:aaaaa-aa";

actor class Main() {
	public func greet(name : Text) : async Text {
		ignore await Management.raw_rand();
		return "Hello, " # name # "!";
	};
};
```

the vscode extension wants the did in a different path though
`file "/Users/moritz/projects/icrc7_launchpad/.dfx/local/lsp/aaaaa-aa.did" does not existMotoko(M0009)`

`dfx 0.16.0`
`motoko extension v0.14.8`

**Comment by rvanasa** [comment]
Here is a workaround which we use in the base library ([source code](https://github.com/dfinity/motoko-base/blob/7939b372f44836049b1afeedeee3a90e3741a2ad/src/Random.mo#L35)):

```motoko
actor class Main() {
    let Management = (actor "aaaaa-aa" : actor {
        raw_rand : () -> async Blob;
    });

    public func greet(name : Text) : async Text {
        ignore await Management.raw_rand();
        return "Hello, " # name # "!";
    };
};
```

**Comment by letmejustputthishere** [comment]
thanks for the workaround. another way of solving this without having to rewrite the management canister interface is to simply copy the `aaaaa-aa.did` to `.dfx/local/lsp`

**Comment by ggreif** [comment]
Some projects just check in the `IC.mo` file like this: https://github.com/sonicdex/motoko-token-canister/blob/main/src/token/Cap.mo#L9

**Comment by letmejustputthishere** [comment]
yes, but then i have to generate the motoko bindings from the did file which adds friction

**Comment by rvanasa** [comment]
The extension now uses a default management canister Candid file as of v0.15.0. Hopefully this resolves the issue in your case; otherwise, let me know and I'll take another look.

**Comment by letmejustputthishere** [comment]
hey @rvanasa, i see your hardcoding the management canister did. is there a way to include it via git submodules or smth similar? i just recently opened a [PR there](https://github.com/dfinity/interface-spec/pull/275) to make working with it a bit more convenient

**Comment by rvanasa** [comment]
Thanks for the link! I'll include a script to generate the Candid file from this URL.
